runtime/src/identity.rs

```diff
@@ -1282,6 +1315,32 @@ impl<T: Trait> Module<T> {
         return None;
     }
 
+    pub fn is_identity_has_valid_kyc(claim_for: IdentityId, buffer: u64) -> bool {
```

Comment:
I wonder if this function should also return the DID of the trusted KYC provider that provided the claim (when result is true, otherwise null if false). This means that the calling function can log that there is a KYC claim, and the issuer of that claim.